client/src/modules/dashboard/Venues/pages/VenuePage.tsx

```diff
@@ -26,6 +48,18 @@ export const VenuePage: NextPageWithLayout = () => {
   if (!data.venue)
     return <NextError statusCode={404} title="Venue not found" />;
 
+  const chapterId = data.venue.chapter.id;
+
+  const clickDelete = async () => {
+    const ok = await confirmDelete({
+      body: 'Are you sure you want to delete this venue? All information related to venue will be deleted. Venue deletion cannot be reversed.',
+      buttonText: 'Delete Venue',
+    });
+    if (!ok) return;
+    deleteVenue({ variables: { venueId, chapterId } });
```

Comment:
```suggestion
    deleteVenue({
      variables: { venueId, chapterId },
      refetchQueries: [
        { query: DASHBOARD_CHAPTER, variables: { chapterId } },
        { query: DASHBOARD_EVENTS },
        { query: DASHBOARD_VENUES },
        ...eventRefetches(data),
      ],
    });
```
Refetching `DASHBOARD_VENUE` is not needed, as it would need to get id of the _no longer existent_ venue.
`DASHBOARD_EVENT` will need to be added for each event that had this venue. See the `eventRefetches` below.